src/api/queryKey.ts

```diff
@@ -57,8 +58,25 @@ const gameQueryKeys = {
   }),
 };
 
+const chatQueryKeys = {
+  chatHistory: (roomId: string) => ({
+    _def: 'chat-history',
+    queryKey: ['chat-history', roomId],
+    queryFn: (): Promise<HttpResponse<ChatHistory>> => fetcher.get(`/v1/chat/${roomId}/messages`),
+    enabled: !!roomId,
+  }),
+
+  chatDmRoomId: (userId: number, friendId: number) => ({
+    _def: 'chat-dm-room-id',
+    queryKey: ['chat-dm-room-id', userId, friendId],
+    queryFn: (): Promise<HttpResponse<{ roomId: number }>> =>
```

Comment:
```suggestion
    queryFn: () =>
```

`fetcher` 함수에 제너럴이 선언되어 있어요. 다른 쿼리키를 참고하셔서 동일하게 변경해주세요.